### DISCUSSION TITLE
手动合并问题

### DISCUSSION THREAD

**Discussion by BondChang** [ymcui/Chinese-LLaMA-Alpaca#583] [Q&A]
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32505512/a2148091-3403-40be-94d5-8978162554d0)
模型合并完有三个.bin目录，请问怎么使用。我用的privategpt，需要指定一个模型的位置

**Comment by ymcui** [comment]
你看一下教程，需要转成privateGPT支持的格式：https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/使用privateGPT进行多文档问答

**Reply by BondChang** [reply]
您好，在这个wiki页面中没找到如何转成privategpt需要的格式的方法，方便标注下吗

**Reply by ymcui** [reply]
wiki里有的，麻烦自行学习：https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/llama.cpp量化部署

**Reply by BondChang** [reply]
好的，谢谢
